api/src/controllers/team.js

```diff
@@ -47,10 +48,18 @@ async function createTeam(req, res, next) {
     // Notify registration via email; don't await completion.
     transporter.sendMail({
       to: email,
+      subject: 'Welcome to Researchify',
       template: 'signup',
       context: {
         name: email,
       },
+      attachments: [
```

Comment:
Could we place the image in the views folder so as to not combine source code with static assets. Maybe create a subfolder within `views/` called `images/` and store the pic there?

Reply:
Was meant to do this but forgot. Thanks for reminding. Done now.